web/viewer.html

```diff
@@ -294,10 +296,6 @@
                 <span id="numPages" class="toolbarLabel"></span>
               </div>
               <div id="toolbarViewerRight">
-                <button id="presentationMode" class="toolbarButton hiddenLargeView" title="Switch to Presentation Mode" tabindex="31" data-l10n-id="presentation_mode">
-                  <span data-l10n-id="presentation_mode_label">Presentation Mode</span>
-                </button>
-
 <!--#if GENERIC-->
                 <button id="openFile" class="toolbarButton hiddenLargeView" title="Open File" tabindex="32" data-l10n-id="open_file">
```

Comment:
Shouldn't these `tabindex`s, for the buttons in the `toolbarViewerRight`-div, be renumbered starting from `31` instead now?
(Since there's also a "hole" in the numbering between the "download" and the "editorFreeText" buttons now.)